cwl/src/main/scala/cwl/InitialWorkDirRequirement.scala

```diff
@@ -44,7 +44,7 @@ final case class StringDirent(
 
 object InitialWorkDirRequirement {
 
-  final type IwdrListingArrayEntry = File :+: Directory :+: StringDirent :+: ExpressionDirent :+: StringOrExpression :+: CNil
+  final type IwdrListingArrayEntry = File :+: Directory :+: ExpressionDirent :+: StringDirent :+: StringOrExpression :+: CNil
```

Comment:
maybe throw a comment in here as to why you did this

Reply:
Sure. FYI, it's because expressions were also getting matched by the string case, so I never saw `Expression` being hit